.github/workflows/lint.yml

```
@@ -6,29 +6,21 @@ jobs:
+    - run: uv run pyrefly check
     - uses: psf/black@stable
       with:
         options: "--check --verbose"
-    - run: pytest
-    - run: pyroma . --min=10
-    - run: |
-        python -m podcast_transcript_convert |& grep -q 'Usage: transcript2json'
\ No newline at end of file
+    - run: uv run pytest --cov=podcast_transcript_convert --cov-report=term-missing
+    - run: uv run pyroma . --min=10
+    - run: uv run python -m podcast_transcript_convert --help | grep -q 'usage: transcript2json'
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)